react/features/base/dialog/components/web/DialogContainer.js

```diff
@@ -20,6 +20,10 @@ class DialogContainer extends AbstractDialogContainer {
      * @returns {ReactElement}
      */
     render() {
+        if (this.props._rawDialog) {
```

Comment:
This could be called "skip transition" since it's exactly what it does. One has to go and check the code for what a "raw" dialog means otherwise.